sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/datasources/subscriptions/SubscriptionsDbConnector.groovy

```diff
@@ -82,7 +83,7 @@ class SubscriptionsDbConnector implements SubscriptionDataSource, SubscribedProj
       
             connection.withCloseable { it ->
               try {
-                int subscriberId = fetchExistingSubscriberId(subscriber)
+                int subscriberId = getSubscriberId(subscriber)
```

Comment:
Again: Optional would have been ideal here to promote good code structure and error handling